lib/src/component/prop_mixins.dart

```diff
@@ -46,10 +46,10 @@ abstract class _$ReactPropsMixin {
 
   // This private field is namespaced to avoid colliding with other classes.
   @Accessor(key: 'children')
-  dynamic _raw$ReactProps$children;
+  ReactNode _raw$ReactProps$children;
 
   /// The children that were passed in to this component when it was built.
-  List<dynamic>? get children {
+  List<ReactNode>? get children {
```

Comment:
Unfortunately this is a breaking change, since consumers will no longer be able to dynamically invoke methods on children (which they probably shouldn't be doing, and instead using proper type checks). 

And I think it might mess up implicit casting in some cases too, not 100% sure.

I'd just be surprised if there aren't some usages out there that would be broken. 

So, I'd say either we revert this change, or if we really wanted to, perhaps we could do some consumer static analysis testing to see if it affects anything.

Reply:
Kicked off [a static analysis check](https://github.com/Workiva/gha-dart/actions/runs/10375376792)

Reply:
The check did not do well. I'll revert this.

Reply:
https://github.com/Workiva/over_react/pull/893/commits/908bfdf84fd0069e8c3cb591a4e59481482e5a35